modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -156,34 +157,53 @@ export async function createSharedDataProof(
 }
 
 /**
- * Creates an Eddsa KeyShare Proof by appending an ed25519 subkey to an armored gpg private key.
+ * Creates a KeyShare Proof based on given algo.
+ *
+ * Creates an EdDSA KeyShare Proof by appending an ed25519 subkey to an armored gpg private key.
```

Comment:
nit: since you mentioned `(auth)` below, put it here as well.   

even smaller nit:  its `ECDSA` not `EcDSA`  (its because in EdDSA it stands for edwards curve, whereas in EC it stands for elliptic curve)